db/db_impl/db_impl.cc

```diff
@@ -3852,7 +3852,7 @@ Status DBImpl::IngestExternalFiles(
         mutex_.Unlock();
         status = AtomicFlushMemTables(cfds_to_flush, flush_opts,
                                       FlushReason::kExternalFileIngestion,
-                                      true /* writes_stopped */);
+                                      true /* writes_stopped */, true /* nonmem_writes_stopped */);
```

Comment:
I think we do not need a new argument. write_stopped could mean that the writes of both write threads are stopped.